mtp_noms_ops/templates/security/accept_or_reject_check.html

```diff
@@ -51,7 +51,9 @@ <h2 class="heading-medium">{% trans 'Credit details' %}</h2>
           <td>
             <a href="{{ check.credit|sender_profile_search_url }}" class="mtp-print-url-hidden">{{ check.credit.sender_name }}</a>
             <br/>
-            {{ check.credit|format_card_number }} &nbsp; {{ check.credit.card_expiry_date }}
+            {{ check.credit|format_card_number }}
```

Comment:
has justin suggested this change? this is not what it looks like in prototype. if so, the list view should be changed to match

Reply:
Yeah, number and expiry should be on the same line.

Reply:
Apologies, I only meant to apply that change to the history table, which does have a line break in place of a non breaking space https://pm-intelligence-tool.herokuapp.com/delayed_capture_history/review_02